src/infra/ucan.js

```diff
@@ -20,41 +20,43 @@ let verify = (req, res, next) => {
   req.invokerAddress = invokerAddress;
   req.contractAddress = contractAddress;
   req.chainId = chainId;
-  if (token && contractAddress) {
-    collaboratorKey({ contractAddress, invokerAddress, chainId })
-      .then((invokerDID) => {
-        if (invokerDID) {
-          ucans.verify(token, {
-            // to make sure we're the intended recipient of this UCAN
-            audience: serviceDID,
-            // capabilities required for this invocation & which owner we expect for each capability
-            requiredCapabilities: [
-              {
-                capability: {
-                  with: { scheme: "storage", hierPart: contractAddress.toLowerCase() },
-                  can: { namespace: "file", segments: ["CREATE"] }
-                },
-                rootIssuer: invokerDID,
-              }
-            ],
-          }).then((result) => {
-            console.log(result);
-            if (result.ok) {
-              req.isAuthenticated = true;
-            }
-            next();
-          });
-        } else {
-          next();
+
+
+  if (!(token && contractAddress)) {
+    req.isAuthenticated = true;
+    next();
+  }
```

Comment:
we don't wan't to authenticate the user when there is no token and contract adress